components/progressbar/stories/template.js

```diff
@@ -48,8 +51,19 @@ export const Template = ({
 				label: indeterminate ? "" : `${value}%`,
 				customClasses: [`${rootClass}-percentage`],
 			}, context)}
-			<div class="${rootClass}-track">
-				<div class="${rootClass}-fill" style="width: ${value}%;"></div>
+
+			<div
+				class="${rootClass}-track"
+				style="--mod-progressbar-track-color: ${staticColor !== "white" ? ifDefined(trackFill) : undefined}"
```

Comment:
I'd recommend a `styleMap` here and not setting `--mod-progressbar-track-color` when it is not needed. Right now this will create `style="--mod-progressbar-track-color: "` in the markup when static white is enabled.

Reply:
`customStyles` seems to work now that I fixed that `background` property! Thanks for pairing on it with me: https://github.com/adobe/spectrum-css/pull/2929/commits/af335d8ba9ffb939f89267a62785e8fbb09e4f43